content/docs/faq-versioning.md

```diff
@@ -1,48 +1,49 @@
 ---
 id: faq-versioning
-title: Versioning Policy
+title: Sürümleme Prensipleri
 permalink: docs/faq-versioning.html
 layout: docs
 category: FAQ
 ---
 
-React follows [semantic versioning (semver)](https://semver.org/) principles.
+React [anlamsal sürümleme (semver)](https://semver.org/lang/tr/) prensiplerini uygulamaktadır.
 
-That means that with a version number **x.y.z**:
+Bu demektir ki sürüm numarası **x.y.z** ile:
 
-* When releasing **breaking changes**, we make a **major release** by changing the **x** number (ex: 15.6.2 to 16.0.0).
-* When releasing **new features**, we make a **minor release** by changing the **y** number (ex: 15.6.2 to 15.7.0).
-* When releasing **bug fixes**, we make a **patch release** by changing the **z** number (ex: 15.6.2 to 15.6.3).
+* **Uyumsuz değişiklikleri** yayınlarken **x** değerini güncelleyerek  **ana sürümü** yayınlarız  (Örneğin: 15.6.2'den 16.0.0'a geçiş).
+* **Yeni özellikleri** yayınlarken **y** değerini güncelleyerek **minör sürümü** yayınlarız (Örneğin: 15.6.2'den 15.7.0'a geçiş).
+* **Hata düzeltmelerini** yayınlarken **z** değerini güncelleyerek **yama sürümünü** yayınlarız (Örneğin: 15.6.2'den 15.6.3'a geçiş).
 
-Major releases can also contain new features, and any release can include bug fixes.
+Ana sürümler de yeni özellikler içerebilir, ve herhangi bir yeni sürüm hata düzeltmelerini bulundurabilir.
 
-### Breaking Changes {#breaking-changes}
+### Uyumsuz değişiklikler {#breaking-changes}
 
-Breaking changes are inconvenient for everyone, so we try to minimize the number of major releases – for example, React 15 was released in April 2016 and React 16 was released in September 2017; React 17 isn't expected until 2019.
+Uyumsuz değişiklikler hepimiz için zahmetli olduğu için ana sürümlerin sayısını olabildiğince az tutmaya çalışıyoruz – örneğin, React 15, Nisan 2016'da, React 16, Eylül 2017'de yayınlanmıştır; React 17'nin 2019'dan önce yayınlanması beklenmemektedir.
 
-Instead, we release new features in minor versions. That means that minor releases are often more interesting and compelling than majors, despite their unassuming name.
+Bunun yerine yeni özellikleri minör sürümlerle yayınlamaktayız. Bu nedenle minör sürümler çoğu zaman ana sürümlerden daha ilgi çekici ve merak uyandırıcı olmaktadır.
 
-### Commitment to Stability {#commitment-to-stability}
+### Stabiliteye Bağlılık {#commitment-to-stability}
```

Comment:
I believe we can translate `stability` as `kararlilik`.

Reply:
I changed it but although I'm native Turkish speaker, `Kararliliga Baglilik` does not make much sense to me in the given context. 😅  Thanks for the feedback.

Reply:
:D  I see your point. Stable is translated as `kararlı` in many local articles/projects including Pardus and Debian websites. See the results, please. : [Google - Kararlı Sürüm](https://www.google.com/search?q=kararl%C4%B1+s%C3%BCr%C3%BCm)

That's why I came up with that translation. What about now? :) You can use your own translation though. That was just a suggestion.

Reply:
Oh I see, makes much sense now. Thank you.